### DISCUSSION TITLE
`showIncomingCalls` and `showOutgoingCalls` don't use location/quickfix list?

### DISCUSSION THREAD

**Discussion by shyun3** [neoclide/coc.nvim#3898] [Q&A] [answered]
Hello, I noticed that `CocAction('showIncomingCalls')` and `CocAction('showOutgoingCalls')` pull up a sidebar to show the results. However, `coc-references` uses a location or quickfix list (depending on the `coc.preferences.useQuickfixForLocations` setting). I find this inconsistent and inconvenient as well, since the key mappings I've defined for the quickfix window don't apply to this bespoke sidebar. 

Is this behavior intentional? Could these be changed to use the quickfix list?

**Comment by fannheyward** [comment, marked as answer]
By default `showIncomingCalls` and `showOutgoingCalls` will show CallHierarchyItem in `:h coc-tree`. You can use `CocAction('incomingCalls')` to get incoming calls results and define a function to display them in location or quickfix list.

**Comment by chemzqm** [comment]
The data is easier to inspect in a tree view, you can bind custom key-mappings for coc-tree.

**Reply by xeycsu** [reply]
Is there a way to change the width of tree view ?  @chemzqm  thx.

**Reply by chemzqm** [reply]
Use autocmd wtih filetype `coctree`
